### PR TITLE
Release Network Operator v23.1.0-rc.1

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

**Review by rollandf — APPROVED** [review]

File: deployment/network-operator/README.md [hunk 1]
```diff
@@ -363,7 +363,7 @@ imagePullSecrets:
 | `ofedDriver.deploy` | bool | `false` | deploy Mellanox OFED driver container                                                                                                                                     |
 | `ofedDriver.repository` | string | `mellanox` | Mellanox OFED driver image repository                                                                                                                                     |
 | `ofedDriver.image` | string | `mofed` | Mellanox OFED driver image name                                                                                                                                           |
-| `ofedDriver.version` | string | `5.8-1.0.1.1.2` | Mellanox OFED driver version                                                                                                                                              |
+| `ofedDriver.version` | string | `5.9-0.5.6.0` | Mellanox OFED driver version                                                                                                                                              |
```
Comment (adrianchiris) [inline deployment/network-operator/README.md:366]: can you also update version in project README ? just to keep things in sync

otherwise LGTM

**Review by adrianchiris — APPROVED** [review]
/lgtm

@e0ne feel free to self merge once comment is addressed

File: deployment/network-operator/README.md (hunk 1 above)
Comment (e0ne) [reply deployment/network-operator/README.md:366]: > can you also update version in project README ? just to keep things in sync

that's a really good catch! We didn't update README for a years

**Review by rollandf — APPROVED** [review]

**Pull request merged**